### ISSUE TITLE
Problem adding new repository

### ISSUE THREAD

**Issue by JoeKolba** [smart-mobile-software/gitstack#163]
I receive the following message when trying to add a new repository:

can only concatenate list (not "str") to list

I originally had this problem when trying to add a user to a group, it was found to be a problem with an extra space in the group file.  

Here is the http post that is failing:
http://servername/rest/repository/

Form-Data:
Name  :  ANBI
csrfmiddlewaretoken  :  D7v6RQjPFykeCxKnvdzwTT4mfk831hat

**Comment by ruthyCP** [comment]
I have the same problem. 
Do you have a fix for this issue?

**Comment by JoeKolba** [comment]
Yeah switched to rhodecode. Completely free and works much better. Gitstack
has a lot of bugs and shouldn't even be offered for free

**Comment by poiuytrez** [comment]
Thank you for your feedback.

Guillaume Charhon
Co-founder & CTO | Arise.io
E: guillaume@arise.io
T: +33 6 41 69 69 37
http://youtu.be/nLNML-60pqM

**Comment by ruthyCP** [comment]
Do you have a solution for this error?
